templates/nginx.yml

```diff
@@ -154,15 +154,15 @@ parameters:
   name: ALLOWED_ORIGIN_DOMAINS
   value: '[".svc.cluster.local"]'
   required: true
-- description: Effective hostname for this service
-  name: FLASK_SERVER_NAME
-  required: true
 - description: URL to Flask service
   name: FLASK_SERVICE_URL
   required: true
 - description: Nginx default server name
   name: NGINX_SERVER_NAME
   required: true
+- description: Nginx server name for the VPN host
+  name: NGINX_VPN_SERVER_NAME
+  required: true
```

Comment:
This cause issue if not set, have we defined them in stage and prod?

Reply:
Yes, I added all the required stuff to the three environments!